.storybook/themes/news.scss

```
@@ -2732,6 +2760,17 @@
 						),
 					),
 				),
+				lead-art-carousel-close-icon: (
+					right: calc(100% - 48px),
```

Is `48px` the same as global-spacing-4 - so can this become - 

```suggestion
					right: calc(100% - var(--global-spacing-4)),
```

---

updated to global spacing